README.md

```diff
@@ -121,10 +122,24 @@ After enabling this option, the following key bindings are available:
 *   `prefix + ctrl + alt + m`: Start monitoring a pane, return it in focus and notify in bash and telegram when it finishes.
 
 Additionally, you can use the `set -g @tnotify-telegram-all 'on'` option to send all notifications to telegram.
```

Comment:
To ensure correct formatting please keep the extra line between the text and the admonition.